sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -535,7 +537,7 @@ def _last_error_message(self):
     else:
       return 'unknown error'
 
-  def wait_until_finish(self):
+  def wait_until_finish(self, duration=None):
```

Comment:
A comment explaining the units for duration (milliseconds?) and that `duration=None` actually means "wait forever" would be helpful. (I find this naming somewhat counter-intuitive, but it's too late to change now.)

Reply:
Agree that the parameter name/default is not optimally chosen. Added a docstring.